src/main/java/blackjack/dto/BettingResultDto.java

```diff
@@ -0,0 +1,14 @@
+package blackjack.dto;
+
+import blackjack.domain.betting.DealerBetting;
+import blackjack.domain.betting.PlayerBetting;
+
+public record BettingResultDto(String name, int betting) {
+   public static BettingResultDto from(final PlayerBetting bettingResult) {
+       return new BettingResultDto(bettingResult.getName().getName(), bettingResult.getBetting());
+    }
+
+    public static BettingResultDto from(final DealerBetting dealerBettingResult) {
+        return new BettingResultDto(dealerBettingResult.getName().getName(), dealerBettingResult.getBettingMoney().getMoeney());
```

Comment:
```suggestion
        return new BettingResultDto(dealerBettingResult.getName().getName(), dealerBettingResult.getBettingMoney().getMoney());
```

메서드명 오타요!

+ 위와 같은 상황에서 getter가 getter를 호출하고 있는데 디미터의 법칙을 생각했을 때 어떤 느낌이 오시나요!

Reply:
`dto는 getter가 암묵적으로 합법적인 허용이 되니까~ `라고 생각하며 디미터 법칙을 크게 고려하지 않았던 것 같네요😂
각 getter를 중첩사용하지 않고 바로 리턴할 수 있도록 수정하겠습니다!